contracts/governance/extensions/IGovernorTimelock.sol

```diff
@@ -11,6 +11,16 @@ import "../IGovernor.sol";
  * _Available since v4.3._
  */
 abstract contract IGovernorTimelock is IGovernor {
+    /**
+     * @dev The proposal hasn't been queued yet.
+     */
+    error GovernorProposalNotQueued(uint256 proposalId);
+
+    /**
+     * @dev The proposal has already been queued.
+     */
+    error GovernorProposalAlreadyQueued(uint256 proposalId);
```

Comment:
I am confused by these names because they seem to go against the guidelines. Is `GovernorProposal` considered to be the domain here? I would have expected `GovernorUnqueuedProposal` and `GovernorAlreadyQueuedProposal` I think.

Reply:
Agree. Thanks for taking the guidelines this seriously.

I went for `GovernorAlreadyQueuedProposal` and `GovernorNotQueuedProposal`. This is because `Unqueued` sounds to me like it was already queued before.